mininet/node.py

```diff
@@ -746,6 +749,15 @@ def __init__(
             dns=self.dns,
         )
 
+        # TODO: remove docker before creating it -- Jinwoo Kim
```

Comment:
We need to check if rm is True, no? Something like:
```suggestion
        if kwargs.get("rm"):
            # do the remove code here
```